frontend/src/components/landing/nav-bar/Navbar.tsx

```diff
@@ -38,7 +38,7 @@ const NavBar = () => {
           </OutboundLink>
           <OutboundLink
             className={styles.link}
-            eventLabel={i18n._(LINKS.guideUrl)}
+            eventLabel={`Navbar / ${i18n._(LINKS.guideUrl)}`}
```

Comment:
There is another NavBar component at `components/common/nav-bar`, need to update the guide link in that too